src/ehrdata/pl/vitessce.py

```diff
@@ -0,0 +1,103 @@
+from __future__ import annotations
+
+from functools import reduce
+from operator import or_, truediv
+from pathlib import Path
+from typing import TYPE_CHECKING
+
+from vitessce import AnnDataWrapper, VitessceConfig
+from vitessce import Component as cm
+
+if TYPE_CHECKING:
+    from lamindb import Artifact
+    from zarr.storage import Store
+
+
+def gen_config(
```

Comment:
what about `gen_vitessce_config`? You think too long or hard to spell? would be more descriptive

Reply:
Then I’d re-export the function directly from `pl`, so there is no repeated namespace: I wouldn’t do `ehrdata.pl.vitessce.gen_vitessce_config`

Reply:
aah my bad. you're right. I like the way it is now, I thought its just `ehrdata.pl.gen_config`, but `ehrdata.pl.vitessce.gen_config` is indeed better than that